test/development/acceptance/__snapshots__/ReactRefreshLogBox-app-doc.test.ts.snap

```diff
@@ -18,7 +18,8 @@ Error:
 
 Caused by:
     0: failed to process input file
-    1: Syntax Error"
+    1: Parser config: Es(EsConfig { jsx: true, fn_bind: false, decorators: false, decorators_before_export: false, export_default_from: false, import_assertions: true, private_in_object: false, allow_super_outside_method: false })
```

Comment:
I added this to help users of `@swc/core`, but I can remove this for `next-swc` if required